.github/workflows/cron-tests.yaml

```diff
@@ -4,6 +4,8 @@ name: Additional tests
     branches:
       - main
       - prerelease_test
+      - trigger/additional_tests
```

Comment:
what is this new branch?

Reply:
@konskov It is a branch that you can use to trigger this workflow. You can push your local changes into our main repository to trigger this workflow using this branch name. 

We have these trigger workflows in other GitHub actions as well (e.g., [here](https://github.com/timescale/timescaledb/blob/main/.github/workflows/memory-tests.yaml#L8) and [here](https://github.com/timescale/timescaledb/blob/main/.github/workflows/sanitizer-build-and-test.yaml#L11))